test/Algebra/Graph/Test/AdjacencyIntMap.hs

```diff
@@ -44,6 +44,9 @@ testAdjacencyIntMap = do
     testDfsForest            t
     testDfsForestFrom        t
     testDfs                  t
+    testBfsForest            t
```

Comment:
Let's keep the order consistent everywhere :)

Reply:
Haha, hopefully I've fixed them all by now